package_extractor.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+import yaml
+import argparse
+from io import open
+
+INTEGRATION = 'integration'
+SCRIPT = 'script'
+
+
+def extract_code(yml_path, output_path, demisto_mock, commonserver=None, yml_type=None):
```

Comment:
I would have splitted this into couple of functions or just added some blank lines for some visual/logical separation, you have a setup stage and then extraction, and creation stages here